tests/unit/test_tasks.py

```diff
@@ -282,18 +282,20 @@ def test_get_hg_hssp(self, mock_exists, mock_bz2file):
         instance = mock_bz2file.return_value
         instance.__enter__.return_value.read.return_value = 'data'
 
-        from xssp_rest.tasks import get_hg_hssp
-        content = get_hg_hssp('TTCCPSIVARSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN')
-
+        from xssp_rest.tasks import mkhssp_from_sequence
+        result = mkhssp_from_sequence.delay('TTCCPSIVARSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN',
+                                            'hssp_stockholm')
```

Comment:
The test is for `hg_hssp` but the type is `hssp_stockholm`. See my other comments.